addons/mail/static/src/components/composer_suggested_partners_list/composer_suggested_partners_list.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.ComposerSuggestedPartnersList" owl="1">
+        <!-- List of followers -->
+        <div class="o_ComposerSuggestedPartnersList">
+
+            <t t-foreach="state.isShowMore ? thread.suggestedPartners : thread.suggestedPartners.slice(0,3)" t-as="recipient">
+                <ComposerSuggestedPartner suggestedPartnerLocalId="recipient.localId" />
+            </t>
+
+            <t t-if="thread.suggestedPartners.length > 3">
+                <a href="#" t-if="!state.isShowMore" t-on-click="_onClickShowMore" class="o_suggested_show_more_less o_suggested_toggle">
+                    Show more
+                </a>
+                <a href="#" t-if="state.isShowMore" t-on-click="_onClickShowLess" class="o_suggested_show_more_less o_suggested_toggle">
```

Comment:
- split `t-if` and `<a>` (use `t-if` in dedicated pseudo-tag `<t>`)
- `class` before `t-on-`
- use convention for classnames, e.g. `o_ComposerSuggestedRecipientList_showLess`
- unused `o_suggested_toggle`
- honestly, would prefer defining these links as `btn-link` instead of `<a>`. It's annoying to need `href="#"` + `preventDefault()`.